dist/index.d.ts

```diff
@@ -498,14 +501,38 @@ interface GPUImageCopyBuffer
 }
 
 interface GPUImageCopyImageBitmap {
```

Comment:
This should be GPUImageCopyExternalImage, but GPUImageCopyImageBitmap should also be included and marked as deprecated, I think.

Reply:
Thanks for catching that. Should be resolved now.

The volume of diffs between the generated file and the dist file make it really easy to miss stuff like this. 😞

Reply:
Unfortunately, yeah. We should be able to get the diffs down over time (see issues filed here https://github.com/darionco/bikeshed-to-ts/issues)